commands/system_e2e_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"time"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlE2ETestSuite) TestGetBusyCmd() {
```

Comment:
These tests are duplicated on https://github.com/mattermost/mmctl/pull/339, right?

Reply:
Haha, yeah. Good catch, I probably and accidentally indexed this file to this branch 🤦‍♂️